### PR TITLE
feat: add controller and pipeline info metrics

### PR DESCRIPTION
**Pull request by whynowy**
1. Added controller build info metrics
2. Added pipeline build info metrics
3. Moved `webhook` package to `./pkg/`

### CODE REVIEW

File: pkg/daemon/server/daemon_server.go [hunk 1]
```diff
@@ -227,6 +236,77 @@ func (ds *daemonServer) newHTTPServer(ctx context.Context, port int, tlsConfig *
 		log.Info("Not enabling pprof debug endpoints")
 	}
 
-	go ds.exposeMetrics(ctx)
 	return &httpServer
 }
+
+// calculate processing lag and watermark_delay to current time using watermark values.
+func (ds *daemonServer) exposeMetrics(ctx context.Context) {
```
Comment (vigith) [inline pkg/daemon/server/daemon_server.go:243]: are we not emitting watermarks for the rest of the vertices?

Comment (vigith) [reply pkg/daemon/server/daemon_server.go:243]: I see that we only have for sources and sinks, did i get that wrong?

Comment (whynowy) [reply pkg/daemon/server/daemon_server.go:243]: We emits the diff of the source and sink, which is the lag we show on the ui, also emits source diff from now.

Comment (whynowy) [reply pkg/daemon/server/daemon_server.go:243]: Maybe we could emit watermark for each vertex directly...

**Review by vigith — APPROVED** [review]

**Pull request merged**